src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaLineItemService.java

```diff
@@ -76,4 +76,11 @@ public LineItem createDuplicate(LineItem originalLineItem, BudgetScenario budget
 
         return this.lineItemRepository.save(lineItem);
     }
+
+    @Override
+    public void deleteMany(List<Integer> lineItemIds) {
+        for (Integer lineItemId: lineItemIds) {
+            this.deleteById(lineItemId);
```

Comment:
Let's optimize this by using: https://stackoverflow.com/questions/36763440/how-do-you-handle-with-bulk-deleting-by-an-array-of-ids-in-spring-data-jpa